client/src/pages/CodeUI.js

```diff
@@ -9,14 +9,16 @@ import {
   Typography,
   Toolbar,
 } from "@material-ui/core/";
+import { useHistory } from "react-router-dom";
+import { UserContext } from "../contexts/UserContext";
 import { withStyles } from "@material-ui/core/styles";
 import { sizing } from "@material-ui/system";
 import "codemirror/lib/codemirror.css";
 import "codemirror/theme/material.css";
 import { theme } from "../themes/theme";
-
+import socket from "../socket/socket";
 require("codemirror/mode/xml/xml");
-require("codemirror/mode/javascript/javascript");
+require("codemirror/mode/python/python");
```

Comment:
@abohakobyan - can you make a new ticket in Zenhub so that we can let users select languages - would be good to work on that soon

Reply:
I think the Run Code Ticket includes this